CleanCode.SolidPrinciples/SingleResponsibility/StringHolidaysLaugh.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CleanCode.SolidPrinciples
+{
+    class StringHolidaysLaugh
```

Comment:
Ta klasa to lekki overkill :) Można  śmiech przechowywać np. w Holiday obiekcie. Ale w  sumie może zostać :)